packages/react-components/src/templates/ProfileResearch.tsx

```diff
@@ -33,72 +34,60 @@ const ProfileResearch: React.FC<ProfileResearchProps> = ({
   skillsDescription,
   questions,
 
-  editBackgroundHref,
   editSkillsHref,
   editQuestionsHref,
 }) => {
   const { isEnabled } = useFlags();
 
-  return (
-    <ProfileCardList>
-      {{
-        card: teams.length
-          ? teams.map((team) => (
-              <ProfileBackground
-                key={team.id}
-                {...team}
-                firstName={firstName}
-              />
-            ))
-          : null,
-        editLink:
-          editBackgroundHref === undefined
-            ? undefined
-            : {
-                href: editBackgroundHref,
-                label: 'Edit role on ASAP',
-                enabled: isEnabled('PROFILE_EDITING'),
-              },
-      }}
-      {{
-        card: skills.length ? (
-          <ProfileSkills
-            skillsDescription={skillsDescription}
-            skills={skills}
-          />
-        ) : null,
-        editLink:
-          editSkillsHref === undefined
-            ? undefined
-            : {
-                href: editSkillsHref,
-                label: 'Edit expertise and resources',
-                enabled: isEnabled('PROFILE_EDITING'),
-              },
-      }}
-      {{
-        card: questions.length ? (
-          <QuestionsSection firstName={firstName} questions={questions} />
-        ) : null,
-        editLink:
-          editQuestionsHref === undefined
-            ? undefined
-            : {
-                href: editQuestionsHref,
-                label: 'Edit open questions',
-                enabled: isEnabled('PROFILE_EDITING'),
-              },
-      }}
-      {{
-        card: (
-          <CtaCard href={createMailTo(email)} buttonText="Contact">
-            <strong>Interested in what you have seen?</strong> <br />
-            Why not get in touch with {displayName}?
-          </CtaCard>
-        ),
-      }}
-    </ProfileCardList>
-  );
+  const cards = [
```

Comment:
If you inlined this exact array though (`<tag>{[...]}</tag>`), surely it'd work?